scripts/resource.js

```diff
@@ -6,21 +6,16 @@ var resources = [];
 function Resource (options){
```

Comment:
you can replace the code inside this function with:
for (key in options) {
this[key] = options[key];
}
so that if you want to add another property to your object you only have to add the property and you don't have to add another line of code (code does it automatically)